### ISSUE TITLE
export terraform bugs: Error: Parse error on "}" (RCURLY) at [1, 448]

### ISSUE THREAD

**Issue by dingxuanyao** [OpsLevel/cli#31]
```
➜  terraform git:(ty/opslevel-to-terraform) ✗ opslevel version  
{
    "version": "2022.04.22",
    "git": "none",
    "go": {
        "version": "go1.17.8",
        "compiler": "gc",
        "os": "darwin",
        "arch": "amd64"
    },
    "opslevel": {
        "app_commit": "3bca3dcb052b",
        "app_version": "2022-06-02_15-03-46+00-00"
    }
}
➜  terraform git:(ty/opslevel-to-terraform) ✗ opslevel export terraform --log-level DEBUG
Writing files to: /Users/tonyyao/workspace/tophat-infrastructure/deployment/terraform/terraform
Error: Parse error on "}" (RCURLY) at [1, 448]
➜  terraform git:(ty/opslevel-to-terraform) ✗ 
```

It looks like the error is happening with `func exportChecks`. i.e. `opslevel_service_*.tf`, `opslevel_teams.tf` + other terraform files are being populated properly, but `opslevel_checks_*.tf` files are empty. `import.sh` also ends with `# Checks`, but none of the import commands.

**Comment by dingxuanyao** [comment]
Also, it looks like the `--log-level` flag is not doing anything

**Comment by rocktavious** [comment]
This was fixed in:  https://github.com/OpsLevel/opslevel-go/pull/53
So it would have been fixed in the [v2022.5.20](https://github.com/OpsLevel/opslevel-go/releases/tag/v2022.5.20) release.
